app/javascript/components/App.js

```diff
@@ -8,15 +8,40 @@ import {
 import Navigation from './components/Navigation'
 import Home from './pages/Home'
 import NotFound from './components/NotFound'
+import ApartmentIndex from './pages/ApartmentIndex'
 
 class App extends React.Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      apartments: []
+    }
+  }
+  
+  componentDidMount(){
+    this.readApart()
+    console.log(this.state)
+  }
+  
+  readApart = () => {
+    fetch("http://localhost:3000/apartments")
```

Comment:
Since you are building a monolithic app you can fetch to just `/apartments` the base url is the same in all places of the app.

Reply:
added the change, thank you